feeds/wlan-ap/opensync/files/bin/wlan_ap_redirector.sh

```diff
@@ -1,11 +1,30 @@
 #!/bin/sh
 
-if [ $# -ne 1 ] ; then
-	echo "Usage: $0 <redirector address>" >&2
-	exit 1
+AP_PRIVATE_KEY_FILE="/usr/opensync/certs/client_dec.key"
+AP_CERTIFICATE_FILE="/usr/opensync/certs/client.pem"
+AP_DEVICE_ID_FILE="/usr/opensync/certs/client_deviceid.txt"
```

Comment:
The DigiCert option depends on the existence of  AP_DEVICE_ID_FILE="/usr/opensync/certs/client_deviceid.txt".
How does this get created?  It does not exist on my CIG WF194c.
Should the code check for the existence of the file and error out instead of doing a call to DigiCert that will fail?

Reply:
This file is part of the AP package that is being created by the PKI scripts as you can see [here](https://github.com/Telecominfraproject/wlan-pki-cert-scripts/blob/digicert/digicert-generate-ap-certs.sh#L72).

Will add a file existence check. :+1:

Reply:
Thanks for the link.  Should this other branch also have a PR or be part of this PR?  Once you add the file check, do you want this PR merged alone?

Reply:
"generate data to be loaded to the AP" <- this

We need to wait for the merge in the PKI scripts repo before we can merge this PR.